src/lib/themes/high-contrast.js

```diff
@@ -73,8 +73,8 @@ const blockColors = {
     // For productionizing: we should look into making this field be the color of the text on the blocks
     // e.g. move and steps in move (10) steps, and rename this property below for controlling the text inside
     // of inputs e.g. 10
```

Comment:
These comments should probably have been removed.

Reply:
Good catch @mxmou! I'll include that in my next color theme pull request.